src/test/v2.0/advanced/fdc3.raiseIntent-NoAppsFound.ts

```diff
@@ -0,0 +1,101 @@
+        await control.raiseIntent("aTestingIntent", "testContextY", {
+          appId: "IntentAppAId",
+          instanceId: "NonExistentInstanceId",
+        });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve1 = "(RaiseIntentFailTargetedAppResolve1) Should fail to raise intent when targeted app intent-a, context 'testContextY', intent 'aTestingIntent' and AppIdentifier IntentAppAId do not correlate";
+    it(RaiseIntentFailTargetedAppResolve1, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY", { appId: "IntentAppAId" });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve2 = "(RaiseIntentFailTargetedAppResolve2) Should fail to raise intent when targeted app intent-a, context 'testContextY', intent 'aTestingIntent' and AppIdentifier NonExistentApp do not correlate";
+    it(RaiseIntentFailTargetedAppResolve2, async () => {
+      try {
+        await control.raiseIntent("aTestingIntent", "testContextY", { appId: "NonExistentApp" });
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    });
+
+    const RaiseIntentFailTargetedAppResolve3 = "(RaiseIntentFailTargetedAppResolve3) Should fail to raise intent when targeted app intent-h, context 'testContextY', intent 'sharedTestingIntent2' and AppIdentifier IntentAppHId do not correlate";
+    it(RaiseIntentFailTargetedAppResolve3, async () => {
+      const { timeout, promise } = sleep(constants.NoListenerTimeout);
+
+      try {
+        await control.raiseIntent("sharedTestingIntent2", "testContextY", { appId: "NonExistentApp" });
+        await promise;
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        clearTimeout(timeout);
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
+      }
+    }).timeout(constants.NoListenerTimeout + 1000);
+
+    const RaiseIntentFailTargetedAppResolve4 = "(RaiseIntentFailTargetedAppResolve4) Should fail to raise intent when targeted app intent-i, context 'testContextY', intent 'sharedTestingIntent2' and AppIdentifier IntentAppIId do not correlate";
+    it(RaiseIntentFailTargetedAppResolve4, async () => {
+      const { timeout, promise } = sleep(constants.NoListenerTimeout);
+
+      try {
+        await control.raiseIntent("sharedTestingIntent2", "testContextY", { appId: "IntentAppIId" });
+        await promise;
+        assert.fail("No error was thrown");
+      } catch (ex) {
+        clearTimeout(timeout);
+        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.NoAppsFound);
```

Comment:
Checking for wrong error
```suggestion
        expect(ex, raiseIntentDocs).to.have.property("message", ResolveError.IntentDeliveryFailed);
```